src/local/butler/reproduce.py

```diff
@@ -132,6 +134,24 @@ def _download_testcase(testcase_id, testcase, configuration):
   return testcase_path
 
 
+def _setup_x():
+  """Start Xvfb and blackbox before running the test application."""
+  environment.set_value('DISPLAY', ':99')
+
+  xvfb_runner = new_process.ProcessRunner('/usr/bin/Xvfb')
+  xvfb_process = xvfb_runner.run(additional_args=[
+      ':99', '-screen', '0', '1280x1024x24', '-ac', '-nolisten', 'tcp'
+  ])
+  time.sleep(5)  # Allow some time for Xvfb to start.
+
+  blackbox_runner = new_process.ProcessRunner('/usr/bin/blackbox')
```

Comment:
Maybe output that starting Blackbox and same for starting Xvfb

Reply:
Done